### DISCUSSION TITLE
Add wheelchair accessible routes?

### DISCUSSION THREAD

**Discussion by rmkenv** [organicmaps/organicmaps#1945] [Ideas]
Would it be possible to add functionality to allow users to view and navigate pedestrian routes that are wheelchair (i.e. ADA) accessible? 

OSM currently has a tag field available where " wheelchair=yes/no/limited/only".  This tag can be applied to various types of point, vector, and/or polygon data. 

See OSM Wiki's Disabilities for more info (https://wiki.openstreetmap.org/wiki/Disabilities#Tagging_for_Disabilities_in_general). It looks like OSM users attempted to do something like this back in 2010 but it is no longer up.  

Proposed workflow and solution: 

1) Add a layer option to allow for users to display vector data where <tag k="wheelchair" v="yes"/>
2) Classify the results by Type to identify routes and entry ways  e.g.   <tag k="type" v="route"/>
3) Optional label to display a route's name or the relation's name

Currently, the University of Maryland Baltimore County Campus has all of it's ADA routes tagged on OSM with <tag k="wheelchair" v="yes"/>

Thoughts?

**Comment by biodranik** [comment]
Are there any products on the market that have this feature? Can we learn from them?

**Comment by rmkenv** [comment]
https://maps.openrouteservice.org/#/place/@-76.70892477035524,39.25368663141432,10

**Comment by pastk** [comment]
https://wheelmap.org/ ?

**Reply by rmkenv** [reply]
Great site, but they only work with points of interest and do not have any route data. 

As per their FAQ's: 

Can I rate roads and sidewalks on Wheelmap.org 
(https://news.wheelmap.org/en/FAQ/)

No – Wheelmap is an online map of places. That is why streets and sidewalks are not rated.
At the moment, this would require a great deal of data e.g. subsoil, exact position of lowered curbs, information about slope and gradient. Collecting this data is very costly and complicated for users. However, as technology advances, we are think that new solutions may well be found and eventually incorporated into Wheelmap in the future.
